examples/chrono_binding.cpp

```diff
@@ -0,0 +1,173 @@
+#include <sqlite_orm/sqlite_orm.h>
+#include <cassert>
+#include <string>
+#include <iostream>
+
+#if __cpp_lib_chrono >= 201907L
```

Comment:
As you are also using Better test for `__cpp_lib_format`, too. You never know what the compiler really implements.
Additionally you need to first include the C++ header, then test for the library feature!

Reply:
done